inntekt/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/inntekt/InntektLøserTest.kt

```diff
@@ -2,28 +2,42 @@
 
 package no.nav.helsearbeidsgiver.inntektsmelding.inntekt
 
-import com.fasterxml.jackson.databind.DeserializationFeature
 import com.fasterxml.jackson.databind.JsonNode
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
-import com.fasterxml.jackson.module.kotlin.readValue
 import io.mockk.every
 import io.mockk.mockk
+import io.mockk.slot
 import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.builtins.MapSerializer
+import kotlinx.serialization.builtins.serializer
+import kotlinx.serialization.json.JsonElement
 import no.nav.helse.rapids_rivers.testsupport.TestRapid
 import no.nav.helsearbeidsgiver.felles.BehovType
+import no.nav.helsearbeidsgiver.felles.HentTrengerImLøsning
 import no.nav.helsearbeidsgiver.felles.InntektLøsning
 import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.Periode
+import no.nav.helsearbeidsgiver.felles.json.fromJson
+import no.nav.helsearbeidsgiver.felles.json.toJson
+import no.nav.helsearbeidsgiver.felles.json.toJsonElement
+import no.nav.helsearbeidsgiver.felles.test.date.februar
+import no.nav.helsearbeidsgiver.felles.test.date.januar
+import no.nav.helsearbeidsgiver.felles.test.mock.mockTrengerInntekt
+import no.nav.helsearbeidsgiver.felles.test.rapidsrivers.sendJson
 import no.nav.helsearbeidsgiver.felles.test.resource.readResource
+import no.nav.helsearbeidsgiver.felles.til
+import no.nav.helsearbeidsgiver.inntekt.ArbeidsInntektInformasjon
+import no.nav.helsearbeidsgiver.inntekt.ArbeidsinntektMaaned
+import no.nav.helsearbeidsgiver.inntekt.Ident
+import no.nav.helsearbeidsgiver.inntekt.Inntekt
 import no.nav.helsearbeidsgiver.inntekt.InntektKlient
 import no.nav.helsearbeidsgiver.inntekt.InntektskomponentResponse
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertNotNull
 import org.junit.jupiter.api.Assertions.assertNull
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.YearMonth
 import java.util.UUID
 
 internal class InntektLøserTest {
```

Comment:
Bruke kotest?

Reply:
was like this when I got here. 
Godt mulig at det er bedre å bruke kotest på kotlin-kode, men synes vi først bør samles og bli enige om hvilket testrammeverk vi skal bruke (og om vi skal bruke tid på å endre eksisterende tester), ikke bytte fram og tilbake gerilja-stil.